packages/super-components/components/msc-advices/src/msc-advices/msc-advices.tsx

```diff
@@ -0,0 +1,155 @@
+
+  @Watch('locale')
+  async updateTranslations() {
+    this.localeStrings = await getTranslations(this.locale);
+  }
+
+  async componentWillLoad() {
+    this.updateTranslations();
+
+    apiClient.aapi
+      .get(`/advices/${this.serviceType}/${this.serviceName}`)
+      .then((response) => {
+        this.adviceGroups = response.data;
+      });
+  }
+
+  private getRecommandationBlock() {
+    return (
+      <>
+        <osds-divider separator />
+        {this.adviceGroups ? (
+          this.adviceGroups.adviceGroups.map((adviceGroup, index) => (
+            <div key={adviceGroup.localizedDescription}>
+              <osds-text
+                class="tile-title"
+                level={ODS_TEXT_LEVEL.body}
+                size={ODS_TEXT_SIZE._200}
+                color={ODS_THEME_COLOR_INTENT.text}
+              >
+                {adviceGroup.localizedDescription}
+              </osds-text>
+              {adviceGroup.advices &&
+                adviceGroup.advices.length > 0 &&
+                adviceGroup.advices.map((advice) => (
+                  <osds-button
+                    key={advice.name}
+                    data-tracking={`${this.dataTrackingPrefix}${advice.name}`}
+                    href={advice.href}
+                    color={ODS_THEME_COLOR_INTENT.primary}
+                    variant={ODS_BUTTON_VARIANT.ghost}
+                  >
+                    <span slot="start">
+                      {advice.localizedName}
+                      {advice.external && (
+                        <osds-icon
+                          class="link-icon"
+                          size={ODS_ICON_SIZE.xxs}
+                          name={ODS_ICON_NAME.EXTERNAL_LINK}
+                          color={ODS_THEME_COLOR_INTENT.primary}
+                        />
+                      )}
+                    </span>
+                    <span slot="end">
+                      <osds-icon
+                        class="link-icon"
+                        size={ODS_ICON_SIZE.xxs}
+                        name={ODS_ICON_NAME.CHEVRON_RIGHT}
+                        color={ODS_THEME_COLOR_INTENT.primary}
+                      />
+                    </span>
+                  </osds-button>
+                ))}
+              {index < this.adviceGroups!.adviceGroups.length - 1 && (
+                <osds-divider separator />
+              )}
+            </div>
+          ))
+        ) : (
+          <osds-skeleton />
+        )}
+      </>
+    );
+  }
+
+  public render() {
+    if (!this.localeStrings) {
+      return (
+        <osds-tile rounded inline>
+          <osds-skeleton />
+        </osds-tile>
+      );
+    }
+
+    return (
+      <Host>
+        <div class="msc-advices-wrapper">
+          <osds-tile
+            class="msc-advices-tile"
+            color={ODS_THEME_COLOR_INTENT.info}
+            variant={ODS_TILE_VARIANT.stroked}
+            checking={true}
```

Comment:
```suggestion
            checking
```

Reply:
Removed, Alexis told me to remove this property